master/struct.html

```diff
@@ -292,79 +292,6 @@ <h3 id="GElement">The <span class='element-name'>'g'</span> element</h3>
 
 <edit:elementsummary name='g'/>
 
-<h2 id="UnknownElement">Unknown elements</h2>
```

Comment:
Instead of completely removing this section, there needs to be text to match browser behavior: unknown elements & their children must not be rendered, and must be represented in the DOM as generic `SVGElement` instances.

There also should be a mention about SVG 1 elements that are now obsolete (see #448). In particular, `<altGlyph>` should be treated as a `<tspan>` rather than disappearing.

Reply:
I didn't tackle this, it seems to need discussion on the text to add. My preference would be to raise that as a separate issue so we can land this PR.

> Instead of completely removing this section, there needs to be text to match browser behavior: unknown elements & their children must not be rendered, and must be represented in the DOM as generic SVGElement instances.

> There also should be a mention about SVG 1 elements that are now obsolete (see #448). In particular, <altGlyph> should be treated as a <tspan> rather than disappearing.

Reply:
Firefox used to treat altGlyph as tspan, we removed all support for altGlyph some time ago though due to lack of use.

Reply:
Now split out into a [separate issue](https://github.com/w3c/svgwg/issues/683)